src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -132,76 +161,153 @@ SDCPUThread::advanceInst(TheISA::PCState next_pc)
 
     const InstSeqNum seq_num = lastCommittedInstNum + inflightInsts.size() + 1;
 
-    DPRINTF(SDCPUThreadEvent, "advanceInst(seq %d, pc %#x, upc %#x)\n",
+    DPRINTF(SDCPUThreadEvent, "advanceInst(seq %d, pc %#x, upc %d)\n",
```

Comment:
This should be changed to issue if that's where the DPRINTF is...

Reply:
Yeah. That was a quick change this morning/last night as I was running out of time to work on this.